lib/operation.ts

```diff
@@ -174,7 +174,7 @@ export class Operation {
     }
     if (map.size === 0) {
       map.set('', null);
-    } else if (map.size === 1) {
+    } else if (map.size === 1 && !this.options.skipJsonSuffix) {
```

Comment:
This is the only change I didn't get. When there's a single content type, why taking into account the skipJsonSuffix flag?
If it is a single content, it should be used without a suffix, regardless of the flag.

Reply:
Well, it could, as long as you don't add another Content Type. If you do, you break the generated code. One could argue that this is a separate subject, but I think if you say Json is default you expect everything that is not Json to have a Suffix. I'd personally think it's the most intuitive behavior in that case as you normally will not have many methods without Json support. If you set Json as default and then it makes sense to me to have the explicit suffix always.
I did a test for that case, were skipJsonSuffix is true and a method only supports Text/Plain. If it had no Suffix I would not see immediately that this is text, not Json.

Reply:
> Well, it could, as long as you don't add another Content Type. If you do, you break the generated code. One could argue that this is a separate subject, but I think if you say Json is default you expect everything that is not Json to have a Suffix. I'd personally think it's the most intuitive behavior in that case as you normally will not have many methods without Json support. If you set Json as default and then it makes sense to me to have the explicit suffix always.
> I did a test for that case, were skipJsonSuffix is true and a method only supports Text/Plain. If it had no Suffix I would not see immediately that this is text, not Json.

I don't think I agree with that. We have, for example, basically a JSON API. However, we have operations to return the content of a image. In that case, even being a single operation with that name, it would be suffixed with `$Image`, which wouldn't be elegant at all. We also have a few methods when creating data that returns the id for the new object. Similarly, in those cases, it would end with `$Text`.

Reply:
Even if you do have a JSON-centric API, it is common to have a method, for example, to get the content of a image. Or a method that returns a generated id as plain text. We have some of those in https://github.com/cyclosproject/cyclos4-ui/blob/master/src/openapi/cyclos-openapi.yaml. In those cases, the single generated method would have a `$Image` or `$Text` suffix, which would be a shame.
To be honest, if there's a single variant, it should have no suffix.

Reply:
Ok, I've updated the PR.